CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+ComplianceAsCode Contribution Guidelines
+===
+
+We aim to create a welcoming environment for all contributors regardless of their age, gender, experience, nationality, or employer. However, to foster smooth collaboration we are asking contributors to adhere to the following rules.
+
```

Comment:
```suggestion

We're so glad you're thinking about contributing to ComplianceAsCode! If you're unsure about anything, just ask -- or submit the issue or pull request to start the conversation! The worst that could happen is you'll politely be asked to change something.

We want to ensure a welcoming environment for all members of the ComplianceAsCode community. Reading the projects CONTRIBUTING policy (you are here), its [LICENSE](https://github.com/ComplianceAsCode/content/blob/master/LICENSE), and its [README](https://github.com/ComplianceAsCode/content/blob/master/README.md), will help you get calibrated.

This project is a public-private partnership with the United States Government as part of its [open source initiatives](https://code.nsa.gov/). The project has welcomed contributions from across the United States Government, private industry, and countries worldwide. Currently, there are four project maintainers who are accountable to ensure a welcoming community and our code of conduct is followed. Should you ever need, please feel free to reach out directly!

- @shawndwells (Shawn Wells // CrowdStrike // shawn.wells@crowdstrike.com)
- @redhatrises  (Gabe Alford // Red Hat // galford@redhat.com)
- @jeffblank (Jeff Blank // U.S. Department of Defense // jeffrey.d.blank@ugov.gov)
- @carlosmmatos (Carlos Matos // Red Hat // cmatos@redhat.com)

```

Reply:
Do we really have to mention the partnership with US Government? It could disturb contributors from other parts of the world, and this partnership is invisible to contributors, or it has been for the past 2 years (i.e. after the shorthand->yaml redesign of the project).

Next, I believe that your list of maintainers is outdated - it contains a person that hasn't contributed to the project for more than five years and isn't even a project member, and four top contributors over the past 2 years are completely missing from the list.
This raises an interesting question how does one become and stay a maintainer. Should it be a title for life?

Reply:
I think that the list of contributors or maintainers (honorable for-life title or not) simply does not belong to Guidelines.

Reply:
Mentioning the US Government is a must, it is the private part of this public/private partnership. It is the reason that CentOS won't have OSPP or STIG profiles as well as for this project to exist and to continue to exist as is. In addition, the project's output get reviewed by the US Government. Code contributions are not the same thing as project ownership and maintainership.

Reply:
This thread is not resolved, please respond to our questions about maintainers.

Reply:
This subject has been also discussed in another thread, and here is the answer: https://github.com/ComplianceAsCode/content/pull/6069#discussion_r517687131